lib/gollum/templates/layout.mustache

```diff
@@ -5,13 +5,13 @@
   <meta name="MobileOptimized" content="width">
   <meta name="HandheldFriendly" content="true">
   <meta name="viewport" content="width=device-width">
-  {{#stylesheet_tag_mustache}}app media: :all{{/stylesheet_tag_mustache}}
-  {{#stylesheet_tag_mustache}}print media: :print{{/stylesheet_tag_mustache}}
+  {{#sprockets_stylesheet_tag}}app{{/sprockets_stylesheet_tag}}
```

Comment:
I noticed that `media: all` is omitted in the new version of this line. Is that because it's a default?